### PR TITLE
feat: add pisces miner

### PR DESCRIPTION
**Pull request by shawaj**
**Issue**

- Link: https://github.com/NebraLtd/hm-pyhelper/issues/130
- Summary: add pisces p100 support

**How**

add pisces p100 which uses i2c bus i2c-0

**Screenshots**

**References**

https://github.com/NebraLtd/hm-pyhelper/issues/130
https://github.com/NebraLtd/hm-pyhelper/issues/131
https://github.com/NebraLtd/hm-pyhelper/issues/133
https://github.com/NebraLtd/helium-miner-software/pull/381
https://github.com/NebraLtd/hm-pktfwd/pull/87
https://github.com/NebraLtd/hm-config/pull/173
https://github.com/NebraLtd/hm-miner/pull/75

**Checklist**

- [x] Tests added
- [x] Cleaned up commit history (rebase!)
- [x] Documentation added
- [x] Thought about variable and method names

### CODE REVIEW

File: .github/workflows/run-update-production.yml [hunk 1]
```diff
@@ -41,14 +42,17 @@ jobs:
         run: |
           gsutil cp docker.config gs://helium-assets.nebra.com/docker.config
           gsutil cp docker.config.rockpi gs://helium-assets.nebra.com/docker.config.rockpi
+          gsutil cp docker.config.rockpi gs://helium-assets.nebra.com/docker.config.pisces
```
Comment (marvinmarnold) [inline .github/workflows/run-update-production.yml:45]: Should be `docker.config.pisces`

Comment (marvinmarnold) [inline .github/workflows/run-update-production.yml:45]: Should be `docker.config.pisces`

File: .github/workflows/run-update-stage.yml [hunk 2]
```diff
@@ -39,14 +40,17 @@ jobs:
         run: |
           gsutil cp docker.config gs://helium-assets-stage.nebra.com/docker.config
           gsutil cp docker.config.rockpi gs://helium-assets-stage.nebra.com/docker.config.rockpi
+          gsutil cp docker.config.rockpi gs://helium-assets-stage.nebra.com/docker.config.pisces
```
Comment (marvinmarnold) [inline .github/workflows/run-update-stage.yml:43]: Should be `docker.config.pisces`

File: .github/workflows/run-update-production.yml (hunk 1 above)
Comment (shawaj) [reply .github/workflows/run-update-production.yml:45]: Oops!

**Comment by shawaj** [comment]
All fixed @marvinmarnold

**Review by marvinmarnold — APPROVED** [review]

**Pull request merged**